remo/reports/forms.py

```diff
@@ -17,7 +18,7 @@ class NGReportForm(happyforms.ModelForm):
         queryset=Activity.active_objects.exclude(name__in=UNLISTED_ACTIVITIES))
     campaign = forms.ModelChoiceField(queryset=Campaign.active_objects.all())
     functional_areas = forms.ChoiceField()
-
+    timezone = forms.CharField()
```

Comment:
could you add a newline after this to make sure we have a newline before the next def?